### DISCUSSION TITLE
rust-course/包根(crate root) 翻译修改建议

### DISCUSSION THREAD

**Discussion by spike014** [sunface/rust-course#461] [内容建议]
`src/main.rs` 和 `src/lib.rs` 即包根(`crate root`) :neckbeard:

参考 `Go` 的 `main.go` 翻译为 包入口文件或者包入口，如何？

**Comment by sunface** [comment]
其实我也一直在纠结这个如何翻译，甚至是不翻译，与此类似的，还有 `package`，因为 crate 翻译成了 包，`package` 就有些难以下手。

总之，感谢你的建议哈哈，先放这里，待我进一步思考后，再回这里讨论下。

**Reply by spike014** [reply]
其实不用太在意翻译一定要信达雅。标新立异，不一直是 `Rust` 给我们的惊喜吗？

```rust
let number = String::from("1");
let str: i32 = 8;
```
在我的代码里，`number` 就是一个字符串，`str` 也不是一个字符串，我的世界我做主。

可以起一个新名字，由他人**传教**，也就是 `rust-course` ，也不负这**圣经**二字。

**Reply by sunface** [reply]
你不做运营可惜了，哈哈，几句话就让我热血沸腾了。。讲得好！

**Reply by spike014** [reply]
谢谢夸奖，继续期待你更好的内容。🤣
